.github/workflows/release-workflow.yml

```diff
@@ -131,8 +133,12 @@ jobs:
           name: linux-artifacts
           path: |
             nexus-staging/
-            samm-cli-linux.tar.gz
-            samm-cli-*.jar
+            samm-cli-${{ env.release_version }}-linux-x86_64.tar.gz
```

Comment:
Since variable interpolation is error-prone, I would prefer to use `${{ github.event.inputs.release_version }}` everywhere. I.e. replace `${{ env.release_version }}` and `$release_version` by it. It's longer, but it will consistently work.